### DISCUSSION TITLE
Philips Hue v2 Dimmer (929002398602) requires scene id 0

### DISCUSSION THREAD

**Discussion by splump** [Koenkk/zigbee2mqtt#22114] [Devices]
It seems it's not possible to save a scene with scene id 0 anymore. The Philips Hue v2 dimmer assumes you can save scene id 0 and scene id 1 in order for the "Hue" button to work with direct bind. Is there some way to get around this in order to be able to switch scenes with the Hue button? The z2m documentation on this device mentions that it should cycle between scene 1 and 2, but according to the debug log this is not the case. It is trying to cycle 0 and 1.

**Comment by wilbowes** [comment]
Also experiencing the same issue with my Philips Hue Dimmer. Had this set up before where I could save a scene within a group with ID 0, however now I'm told the scene must be 1 or higher. Philips Hue Dimmer will only cycle through scene 0 or 1, so I can recall scene 1 but not recall 0 or 2.
